### ISSUE TITLE
PACF and ACF with type = "partial" discrepancy?

### ISSUE THREAD

**Issue by AEBilgrau** [tidyverts/feasts#98]
Am I wrong for expecting that `PACF(<...>)` and `ACF(<...>, type = "partial")` should be identical? From the documentation I do not see any difference.

Off the bat, it seems to be an index off-by-one error as the underlying implementations agree.

Consider:

```r
library(tsibble)
library(tsibbledata)
library(dplyr)

vic_elec %>% feasts::ACF(Temperature, type = "partial") %>% print(n = 3)
## A tsibble: 46 x 2 [30m]
#    lag    acf
#  <lag>  <dbl>
#1   30m -0.395   # <-- ?
#2   60m -0.220
#3   90m -0.141
## … with 43 more rows
vic_elec %>% feasts::PACF(Temperature) %>% print(n = 3)
## A tsibble: 47 x 2 [30m]
#    lag   pacf
#  <lag>  <dbl>
#1   30m  0.994   # <-- ?
#2   60m -0.395
#3   90m -0.220
## … with 44 more rows
```
Compare with:

```r
x <- acf(ldeaths, type = "partial"); print(x)
y <- pacf(ldeaths); print(y)
```

which agree with each other.

**Comment by AEBilgrau** [comment]
Looking that the code, the problem is, that `acf` with `type = "partial"` automatically removes the first index, and `ACF` also drops the second.

**Comment by mitchelloharawild** [comment]
`...` isn't passed to the underlying method, rather it is used to specify the variable used in the ACF or PACF. http://feasts.tidyverts.org/reference/ACF.html

Related discussion around using `...` can be found in #2.

**Comment by AEBilgrau** [comment]
I can try do a fix?

**Comment by mitchelloharawild** [comment]
I'm inclined to remove the `type="partial"` option for `ACF()` and encourage users to use `PACF()` instead. Thoughts?

**Comment by AEBilgrau** [comment]
Well, I guess its nice that you follow the base `acf` and `pacf` and have that 1-to-1 correspondence. Are there any difference as to speed, edge-cases, etc?

**Comment by mitchelloharawild** [comment]
I don't know of any differences. Looking at the code for `stats::acf()`, when `type="partial"` is used it simply calls `pacf()`. My suspicion is that `pacf()` was added after `acf()`.

If you'd like to add a check for `type != "partial"` before removing the first observation in `compute_acf`, then a PR would be great :+1:.

**Comment by AEBilgrau** [comment]
I did just that though it was slightly more complicated as the default type is a vector of length 3 (the arg.match apparently happens later).

**Comment by mitchelloharawild** [comment]
Resolved by #99, thanks!
